Document/0x05c-Reverse-Engineering-and-Tampering.md

```diff
@@ -918,7 +918,7 @@ DDMS also offers a convenient heap dump button that will dump the Java heap of a
 
 Moving down a level in the OS hierarchy, you arrive at privileged functions that require the powers of the Linux kernel. These functions are available to normal processes via the system call interface. Instrumenting and intercepting calls into the kernel is an effective method for getting a rough idea of what a user process is doing, and often the most efficient way to deactivate low-level tampering defenses.
 
-Strace is a standard Linux utility that monitors interaction between processes and the kernel. The utility is not included with Android by default, but can easily be built from source via the Android NDK. Strace is a very convenient way to monitor a process' system calls. Strace depends, however on the `ptrace` system call to attach to the target process, so it only works up to the point at which anti-debugging measures start up.
+Strace is a standard Linux utility that monitors the interaction between processes and the kernel. The utility is not included with Android by default, but can easily be built from source via the Android NDK. Strace is a very convenient way to monitor a process's system calls. Strace depends, however on the `ptrace` system call to attach to the target process, so it only works up to the point at which anti-debugging measures startup.
```

Comment:
According to the Cambridge dictionary it should be "start up".
https://dictionary.cambridge.org/dictionary/english/start-up-something

Startup (or start-up) means a small company. https://dictionary.cambridge.org/dictionary/english/start-up

```suggestion
Strace is a standard Linux utility that monitors the interaction between processes and the kernel. The utility is not included with Android by default, but can easily be built from source via the Android NDK. Strace is a very convenient way to monitor a process's system calls. Strace depends, however on the `ptrace` system call to attach to the target process, so it only works up to the point at which anti-debugging measures start up.
```

Reply:
Actually it seems to be correct on a technical context, see here: 

https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/start-up-startup

https://developer.android.com/topic/performance/vitals/launch-time

However, please take a look at the suggestion below.

Reply:
Ok, then let's leave it as it is. Makes sense to me.  Thanks for researching on this!